projects/packages/backup/src/js/components/Admin.js

```diff
@@ -168,7 +169,26 @@ const Admin = () => {
 		return (
 			<Container horizontalSpacing={ 3 } horizontalGap={ 3 }>
 				<Col lg={ 6 } md={ 4 }>
-					<h2>{ __( 'Your cloud backups', 'jetpack-backup-pkg' ) }</h2>
+					<h2>{ __( 'Restore points created with every edit', 'jetpack-backup-pkg' ) }</h2>
+					<p className="jp-realtime-note">
+						{ createInterpolateElement(
+							__(
+								'No need to run a manual backup before you make changes to your site. <ExternalLink>Learn more</ExternalLink>',
+								'jetpack-backup-pkg'
+							),
+							{
+								ExternalLink: (
+									<ExternalLink
+										href={ getRedirectUrl( 'jetpack-blog-realtime-mechanics' ) }
+										target="_blank"
+										rel="noopener noreferrer"
+									/>
```

Comment:
`ExternalLink` component already adds `target="_blank"` and appropriate `rel` prop to the underlying anchor tag.

```suggestion
									<ExternalLink href={ getRedirectUrl( 'jetpack-blog-realtime-mechanics' ) } />
```

Reply:
Ahh yes you're right, excellent Manzoor! Thank you!!  😄 🙌